tests/lava/proc/dense/test_stdp_sim.py

```diff
@@ -550,7 +549,7 @@ def test_rstdp_floating_point(self):
 
         np.testing.assert_almost_equal(weight_before_run, weights_init)
         np.testing.assert_almost_equal(
-            weight_after_run, np.array([[33.4178762]])
+            weight_after_run, np.array([[33.4210359]])
```

Comment:
Why does this have to be changed? Was the unit test incorrect?

Reply:
As we reduced the constant of `2**32` to `2**16` above, the values of these tests change slightly. We had to change those constants as Windows was not able to use them.